v63004/u_inref/gtm8909.exp

```diff
@@ -0,0 +1,73 @@
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+set timeout 60
+spawn /usr/local/bin/tcsh -f
+
+expect_after {
+	timeout { timeout_procedure }
+}
+
+proc timeout_procedure { } {
+	puts "timeout occurred"
+	exit -1
+}
+
+puts "# Expect the shell prompt"
+expect -exact ">"
+puts "# Start LKE help facility"
+send -- "\$gtm_dist/lke\r"
+expect -exact "LKE>"
+send -- "help\r"
+puts "# Expect Help Facility Prompt"
+expect  "Topic?"
+puts "#<Ctrl-C>"
+send -- "\x03\r"
+send -- "\r"
+puts "# Exit LKE>"
+expect -exact "LKE>"
+send -- "exit\r"
+
+puts "# Expect the shell prompt"
+expect -exact ">"
+puts "# Start DSE help facility"
```

Comment:
The script flow is the same for LKE, DSE and MUPIP. It would be nice (not necessary) to fold this into a loop (for or while etc.) so we avoid code duplication. I leave the choice to you.